1-hello-world/Gemfile.lock

```diff
@@ -116,6 +123,9 @@ GEM
       thread_safe (~> 0.1)
     tzinfo-data (1.2015.6)
       tzinfo (>= 1.0.0)
+    websocket-driver (0.6.3)
```

Comment:
I assume this is for `capybara/poltergeist` (see [here](https://github.com/teampoltergeist/poltergeist/blob/master/poltergeist.gemspec#L24))